.github/workflows/pr-reviewed-trigger.yml

```diff
@@ -0,0 +1,42 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Pull Request Reviewed
```

Comment:
We're starting to have quite a few actions. I think it would be good to add the `description` field to all of them to clearly state their role, how they work, and the potential relationships between each others.

Reply:
I don't think there is a description field for the workflow files. I've been adding comments here and there to help people navigate the workflow files. I think `name` is as good as we get unfortunately. 

Here's the docs: https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#about-yaml-syntax-for-workflows

And a similar question on SO: https://stackoverflow.com/questions/70941956/description-for-a-workflow-in-github-actions

Reply:
My bad I was looking at https://docs.github.com/en/actions/sharing-automations/creating-actions/metadata-syntax-for-github-actions#description which is not available for workflows.

So let's use comments to describe the workflows. Maybe it's only me (I'm not super familiar with GitHub Actions) but looking at the files (without reading the PR), it's not immediately obvious what each one does.

Reply:
I just added some docs to the README in the .github/workflows directory. PTAL

Reply:
Thanks that's very helpful!